deployments/basesepolia_addresses.json

```diff
@@ -2,6 +2,9 @@
   "name": "basesepolia",
   "chainId": "84532",
   "addresses": {
-    "VTreasuryV8": "0x07e880DaA6572829cE8ABaaf0f5323A4eFC417A6"
+    "VTreasuryV8": "0x07e880DaA6572829cE8ABaaf0f5323A4eFC417A6",
+    "XVSStore": "0x0ff9Ec1118f1E332CF4DE2A685110316af60F9fD",
+    "XVSVaultProxy": "0xA0D5C7FD3c498ea0a0FDeBaDe3a83D56DA8E2356",
```

Comment:
The XVS token is wrongly set here. According to [this](https://github.com/VenusProtocol/token-bridge/pull/44/files#diff-748fb99ac7a67ecb3fb81cc6f11a691b8241938e6be37bf0c5d633682012029a), it should be `0xE657EDb5579B82135a274E85187927C42E38C021`, but it's `0xe2FC0F0Fa243b1Ca177eb6E023e351583C26E2C4`